core/src/main/java/ai/timefold/solver/core/impl/domain/variable/declarative/ParentVariableType.java

```diff
@@ -0,0 +1,35 @@
+package ai.timefold.solver.core.impl.domain.variable.declarative;
+
+public enum ParentVariableType {
```

Comment:
I think it would be a good idea to explain here the concept of directionality, and how it connects to loops etc.